aiidalab/app.py

```diff
@@ -125,7 +125,6 @@ def from_id(cls, app_id, registry_entry=None, apps_path=None):
 
         return cls.from_registry_entry(path=app_path, registry_entry=registry_entry)
 
-    @property
```

Comment:
Why did you change this?

To me, `is_registered` is an immutable property of the app (unlike `is_installed` or `is_detached` that may change due to user action) so it makes sense for it to be `@property`

Reply:
Okay. I don't have a strong feeling about this. Just in general I would like to minimize unrelated changes in PRs, since it makes the review harder and because Python is not a compiled language there's always a risk of introducing bugs if one is not careful. It's best if refactoring like this were done separately.

Reply:
> It's best if refactoring like this were done separately.

Okay, Will do it next time.